modules/preview-react/select/lib/SelectBase.tsx

```diff
@@ -261,7 +281,11 @@ const SelectWrapper = styled('div')<Pick<SelectBaseProps, 'grow' | 'disabled'>>(
 );
 
 const defaultRenderOption: RenderOptionFunction = option => {
-  return <div>{option.label}</div>;
+  return <div>{defaultRenderSelectedOption(option)}</div>;
+};
+
+const defaultRenderSelectedOption: RenderSelectedOptionFunction = option => {
+  return option.label;
```

Comment:
I kept the default render methods separate to keep the underlying DOM unchanged. I am open to combining them into one default render method, but I was not sure if this is recommended

Reply:
I can't recall why I wrapped the option label in a `<div>` in `defaultRenderOption` 🤔, but it does make sense to preserve the existing DOM structure.